aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -237,6 +241,10 @@ spec aptos_framework::account {
         let new_auth_key = from_bcs::deserialize<address>(new_auth_key_vector);
 
         aborts_if curr_auth_key != new_auth_key && table::spec_contains(address_map, new_auth_key);
+        ensures new_scheme == ED25519_SCHEME || new_scheme == MULTI_ED25519_SCHEME;
```

Comment:
Same as above since `update_auth_key_and_originating_address_table` is called in `rotate_authentication_key_with_rotation_capability`.

Reply:
No need to ensure this.

Reply:
fixed